api/src/main/java/com/viaversion/viaversion/api/protocol/version/VersionRange.java

```diff
@@ -68,6 +68,21 @@ public int getRangeFrom() {
      *
      * @return highest included minor version
      */
+    public int rangeTo() {
+        return rangeTo;
+    }
+
+    @Deprecated/*(forRemoval = true)*/
```

Comment:
Why commenting out the (forRemoval = true) either add it or don't. That's inconsistent.